config/index.js

```diff
@@ -334,6 +334,32 @@ var conf = convict({
       env: 'SUPPORTED_LANGS'
     }
   },
+  redis: {
+    enabled: {
+      default: true,
```

Comment:
Yeah, if it's straightforward enough, let's make sure this runs by default in local dev. It is (or will be) core functionality for the content server.